src/app/content/search/components/SearchResultsSidebar/SearchResultsBarWrapper.tsx

```diff
@@ -125,6 +125,18 @@ export class SearchResultsBarWrapper extends Component<ResultsSidebarProps> {
 
   public componentDidUpdate() {
     this.scrollToSelectedPage();
+
+    const searchSidebar = this.searchSidebar.current;
+
+    if (!searchSidebar) {
+      return;
+    }
+
+    const sidebarChildren = searchSidebar.children;
+    if (sidebarChildren.length === 2) {
```

Comment:
did you try setting `Styled.NavOl` to fill the remaining height with flexbox or something and setting that to overflow:auto?

Reply:
no :dizzy_face: one sec